.github/workflows/publish.yaml

```diff
@@ -5,6 +5,7 @@ name: Publish
 on:
   pull_request:
     branches: [ master ]
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
What's this?

Reply:
oh...I get it...

Reply:
That lets the publish bot react to label changes for a PR (the validation can be configured a bit via adding a `publish-ignore-warnings` label to a PR).